Frends.AzureBlobStorage.ListBlobsInContainer/Frends.AzureBlobStorage.ListBlobsInContainer/Definitions/Source.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+namespace Frends.AzureBlobStorage.ListBlobsInContainer.Definitions
+{
+    /// <summary>
+    ///     Source values.
+    /// </summary>
+    public class Source
+    {
+        /// <summary>
+        ///     Authentication method to use when connecting to Azure Storage. Options are connection string and SAS Token.
+        /// </summary>
+        public AuthenticationMethod AuthenticationMethod { get; set; }
+
+        /// <summary>
+        ///     The base URI for the storage container.
+        /// </summary>
+        [UIHint(nameof(AuthenticationMethod), "", AuthenticationMethod.Sastoken)]
+        [DefaultValue("https://storageaccount.blob.core.windows.net/")]
+        [DisplayFormat(DataFormatString = "Text")]
```

Comment:
You don't need this line, since the property is string-type, so Frends will automatically know that the input type is text.